src/spacy/fortex/spacy/spacy_processors.py

```diff
@@ -351,9 +351,8 @@ def default_configs(cls) -> Dict[str, Any]:
 
         """
         return {
-            "medical_onto_type": "ftx.medical.clinical_ontology."
-            + "MedicalEntityMention",
-            "umls_onto_type": "ftx.medical.clinical_ontology.UMLSConceptLink",
+            "medical_onto_type": "ftx.onto.clinical" + ".MedicalEntityMention",
```

Comment:
We can just merge both the substrings into a single string, since they are now on the same line. Same for line 474

Reply:
yes, all changed according to your comments.